libpod/define/container_inspect.go

```diff
@@ -157,7 +157,7 @@ type InspectMount struct {
 	// "volume" and "bind".
 	Type string `json:"Type"`
 	// The name of the volume. Empty for bind mounts.
-	Name string `json:"Name,omptempty"`
+	Name string `json:"Name,omitempty"`
```

Comment:
@Luap99 Note that this is similar

Reply:
This one is fine, though, `omitempty` is a language keyword, not directly part of the schema.